### PR TITLE
add layerzero docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/481

Hyperlane translations are in a separate PR (due to timing)

### Checklist

- [ ] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [ ] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [ ] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

n/a

### CODE REVIEW

**Review by albertov19 — APPROVED** [review]
LGTM! Please wait until Xinxin reviews this

File: builders/interoperability/protocols/layerzero.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# LayerZero协议
+
+![LayerZero Moonbeam banner](/images/builders/integrations/bridges/layerzero/layerzero-banner.png)
+
+## 概览 {: #introduction }
+
+[LayerZero](https://layerzero.network/){target=_blank}为Web3提供安全的全链（Omnichain）互操作性。LayerZero依赖于Oracle（预言机）和Relayer（中继器）两方在链与链之间提供安全消息传输。LayerZero的基础设施使dApp用户能够通过一键操作与任何连接链上的任何资产或应用程序进行交互。
+
+LayerZero是一个传输层，能够通过低级通信原语实现资产传输。使用LayerZero的DApp被称为用户应用程序，其消息由中继器跨链中继，并由预言机的区块头在目标链上进行验证。查看下方技术堆栈图及其[概念文档](https://layerzero.gitbook.io/docs/faq/messaging-properties){target=_blank}以获取更多信息。
```
Comment (wuxinxin2000) [inline builders/interoperability/protocols/layerzero.md:14]:
```suggestion
LayerZero是一个传输层，能够通过低级通信原语实现资产传输。使用LayerZero的DApp被称为用户应用程序，其消息由中继器跨链中继，并由预言机的区块头在目标链上进行验证。请查看下方技术栈图及其[概念文档](https://layerzero.gitbook.io/docs/faq/messaging-properties){target=_blank}以获取更多信息。
```

File: builders/interoperability/protocols/layerzero.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+![LayerZero Technology Stack diagram](/images/builders/integrations/bridges/layerzero/layerzero-1.png)
+
+LayerZero API为开发Web3应用提供了丰富的套件，确保开发者拥有构建所需的工具。借助这些工具和API，开发者可以使用LayerZero协议及其API，来编写可以轻松部署在与所有LayerZero连接的生态系统dApp。
```
Comment (wuxinxin2000) [inline builders/interoperability/protocols/layerzero.md:18]:
```suggestion
LayerZero API为开发Web3应用提供了丰富的套件，确保开发者拥有构建所需的工具。借助这些工具和API，开发者可以使用LayerZero协议及其API，来编写可以轻松部署在与所有LayerZero连接的生态系统的dApp。
```

File: builders/interoperability/protocols/layerzero.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+--8<-- 'text/disclaimers/third-party-content-intro.md'
+
+## 开始操作 {: #getting-started }
+
+开始使用LayerZero构建跨链应用程序可能所需资源：
```
Comment (wuxinxin2000) [inline builders/interoperability/protocols/layerzero.md:24]:
```suggestion
开始使用LayerZero构建跨链应用程序可能所需的资源：
```

File: builders/interoperability/protocols/layerzero.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- **[开发者文档](https://layerzero.gitbook.io/docs/){target=_blank}** - 技术性指南
+- **[Stargate](https://stargate.finance/){target=_blank}** - 使用LayerZero的桥接UI，由Stargate团队构建
+
+## 合约 {: #contracts }
+
+看部署至Moonbeam的LayerZero合约列表，以及通过LayerZero连接至Moonbeam的网络。
```
Comment (wuxinxin2000) [inline builders/interoperability/protocols/layerzero.md:31]:
```suggestion
查看部署至Moonbeam的LayerZero合约列表，以及通过LayerZero连接至Moonbeam的网络。
```

**Review by wuxinxin2000 — APPROVED** [review]

File: builders/interoperability/protocols/layerzero.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: 通过LayerZero进行跨链操作
+description: 学习如何使用LayerZero协议和API桥接资产以及将Moonbeam dApp连接到多个区块链上的资产和功能的其他方法。
+---
+
+# LayerZero协议
+
+![LayerZero Moonbeam banner](/images/builders/integrations/bridges/layerzero/layerzero-banner.png)
```
Comment (eshaben) [inline builders/interoperability/protocols/layerzero.md:8]:
```suggestion
![LayerZero Moonbeam banner](/images/builders/interoperability/protocols/layerzero/layerzero-banner.png)
```

File: builders/interoperability/protocols/layerzero.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+## 概览 {: #introduction }
+
+[LayerZero](https://layerzero.network/){target=_blank}为Web3提供安全的全链（Omnichain）互操作性。LayerZero依赖于Oracle（预言机）和Relayer（中继器）两方在链与链之间提供安全消息传输。LayerZero的基础设施使dApp用户能够通过一键操作与任何连接链上的任何资产或应用程序进行交互。
+
+LayerZero是一个传输层，能够通过低级通信原语实现资产传输。使用LayerZero的DApp被称为用户应用程序，其消息由中继器跨链中继，并由预言机的区块头在目标链上进行验证。请查看下方技术栈图及其[概念文档](https://layerzero.gitbook.io/docs/faq/messaging-properties){target=_blank}以获取更多信息。
+
+![LayerZero Technology Stack diagram](/images/builders/integrations/bridges/layerzero/layerzero-1.png)
```
Comment (eshaben) [inline builders/interoperability/protocols/layerzero.md:16]:
```suggestion
![LayerZero Technology Stack diagram](/images/builders/interoperability/protocols/layerzero/layerzero-1.png)
```

**Pull request merged**